public/static/docs/command-reference/import.md

```diff
@@ -1,7 +1,7 @@
 # import
 
-Download a file or directory from any <abbr>DVC repository</abbr> (e.g. hosted
-on GitHub) into the <abbr>workspace</abbr>. This also creates a
+Download a file or directory from any Git or <abbr>DVC repository</abbr> (e.g.
+hosted on GitHub) into the <abbr>workspace</abbr>. This also creates a
```

Comment:
Let's just copy from https://dvc.org/doc/command-reference/get:

`Download a file or directory from any <abbr>DVC project<abbr> or Git repository (e.g. hosted on GitHub) into the current working directory.`